caom2repo/caom2repo/core.py

```diff
@@ -500,16 +753,21 @@ def update(self, observation, **kwargs):
         else:
             observation_writer.write(observation, sys.stdout)
     elif args.cmd == 'update':
-        logging.info("Update")
+        logger.info("Update")
         obs_reader = ObservationReader()
         # TODO not sure if need to read in string first
         client.post_observation(obs_reader.read(args.observation))
     else:
-        logging.info("Delete")
+        logger.info("Delete")
         client.delete_observation(collection=args.collection,
                                   observation_id=args.observationID)
 
-    logging.info("DONE")
+    lp = threading.Thread(target=logger_thread, args=(queue,))
+    lp.start()
```

Comment:
Just wondering if you shouldn't do this before starting the processes.

Reply:
The processes have to be started before the logger_thread.